gluecodium/src/test/resources/smoke/comments/output/lime/smoke/DeprecationComments.lime

```diff
@@ -41,4 +41,13 @@ interface DeprecationComments {
         @Deprecated("Unfortunately, this property's setter is deprecated.\nUse [comments.SomeProperty.set] instead.")
         set
     }
+    // Describes the property but not accessors.
+    // @get Gets the property but not accessors.
+    @Deprecated("Will be removed in v3.2.1.")
+    property PropertyButNotAccessors: String {
+        @Deprecated("Will be removed in v3.2.1.")
```

Comment:
Shouldn't text be different for get/set to cover case when message is different?

Reply:
This use case is already covered by previously existing tests.